test/e2e/performanceprofile/functests/2_performance_update/unit_report_performance_update.xml

```diff
@@ -0,0 +1,88 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
is this file suppose to be committed or is it a generate file that we should not add here?
In addition, if this is a generated file it would be advised to change its output path